### PR TITLE
Fix up .stylish-haskell.yaml

### PR DESCRIPTION
**Pull request by ivan-m**
The `records` attribute is no longer valid.

Not sure if this is quite the config you want to use (as it doesn't
match all the code present).

### CODE REVIEW

**Comment by alpmestan** [comment]
Someone else might want to chime in, as I personally am not a huge fan (nor user) of stylish-haskell. If nobody does, let's just merge it.

File: .stylish-haskell.yaml [hunk 1]
```diff
@@ -67,6 +167,18 @@ steps:
...
+# preferred format here.
+#
+# - native: Native newline format. CRLF on Windows, LF on other OSes.
+#
+# - lf: Convert to LF ("\n").
+#
+# - crlf: Convert to CRLF ("\r\n").
+#
+# Default: native.
+newline: native
```
Comment (phadej) [inline .stylish-haskell.yaml]: `lf`, we don't want windows users screwing up things.

**Comment by 3noch** [comment]
If not everyone uses this then contributors will either,

  a) ignore it and get the styling wrong
  b) use it, but make big diffs because not everyone uses it.

I personally use stylish Haskell and like it. But I've also been bitten by it while contributing to other projects.

**Comment by ivan-m** [comment]
Either the config needs to be updated or removed; currently it means automated application keeps breaking.

**Comment by 3noch** [comment]
Oh right. I neglected to notice that this was an update, not a new file. Right on.

**Comment by ivan-m** [comment]
I missed #680 when I wrote this, so one of these should be closed (on phone so can't easily inspect differences).

**Comment by jkarni** [comment]
Closing after #680 got merged (let us know if there are things you think are still missing). Sorry for the slow turnaround.

**Pull request closed without merging**